hw/templates/user_wrapper_tmplt.txt

```diff
@@ -1023,8 +1140,8 @@ module design_user_wrapper_{{ c_reg }} (
         .axis_rrsp_send(axis_rdma_send_rsp_int),
     {% endif %}
     {% if cnfg.en_tcp %}
-        .axis_tcp_recv(axis_tcp_recv_int),
-        .axis_tcp_send(axis_tcp_send_int),
+        .axis_tcp_recv(axis_tcp_sink),
+        .axis_tcp_send(axis_tcp_src),
```

Comment:
To keep it consisten with other interfaces (host, network, RDMA), can we also pass axis_tcp_sink_int and axis_tcp_src_int and have a register stage above? The same way we handle all the other interfaces.